content-test/addon/MetadataParser.test.js

```diff
@@ -0,0 +1,72 @@
+/* globals assert, require */
+"use strict";
+const createMetadataParser = require("inject!addon/MetadataParser");
+const TEST_HTML = `<html>
+  <head>
+    <title>Test Page</title>
+    <link rel="shortcut icon" type="image/x-icon" href="//www.foo.com/icon.png" />
+    <meta property="og:image" content="//www.foo.com/image.png" />
+  </head>
+ </html>`;
+
+describe("MetadataParser", () => {
+  let metadataParser;
+  function setup() {
+    const mockPreviewProvider = {
+      processLinks(link) {
```

Comment:
Removing this becuase we can remove the dependency on preview provider altogether.